src/service/video-manager-impl.js

```diff
@@ -396,6 +440,13 @@ class VideoEntry {
 
     this.hasAutoplay = element.hasAttribute(VideoAttributes.AUTOPLAY);
 
+    const fsOnLandscapeAttr = element.getAttribute(
+        VideoAttributes.FULLSCREEN_ON_LANDSCAPE
+    );
+
+    this.hasFullscreenOnLandscape = fsOnLandscapeAttr !== undefined
```

Comment:
`undefined` check not needed (`getAttribute` actually returns null). Also inconsistent `==` and `===` (just use `==` compiler takes care of it).